client/a8c-for-agencies/sections/marketplace/hosting-overview-v3/hosting-content/premier-agency-hosting/pressable-plan-section/index.tsx

```diff
@@ -245,7 +246,13 @@ export default function PressablePlanSection( {
 
 				<span className="pressable-plan-section__details-footnote">
 					{ translate(
-						`*If you exceed your plan's storage or traffic limits, you will be charged $0.50 per GB and $8 per 10K visits per month.`
+						`*If you exceed your plan's storage or traffic limits, you will be charged %(storageCharge)s per GB and %(trafficCharge)s per 10K visits per month.`,
```

Comment:
> per 10K visits

I think let's also pass `10K` through `numberFormatCompact` - it should render as `10 χιλ.` in EL/Greek.